overlord/snapshotstate/snapshotstate.go

```diff
@@ -330,6 +343,16 @@ func Import(ctx context.Context, st *state.State, r io.Reader) (setID uint64, sn
 		if dupErr, ok := err.(backend.DuplicatedSnapshotImportError); ok {
 			st.Lock()
 			defer st.Unlock()
+			// XXX: ideally, we should retry import.
+			if err := checkSnapshotConflict(st, dupErr.SetID, "forget-snapshot"); err != nil {
+				return 0, nil, err
+			}
+
+			// This is not technically needed since we're keeping state lock;
+			// it's done only for symmetry with export.
```

Comment:
I wouldn't do this, it simplifies also other places that then don't need to list import-snapshot in their conflicts